groundhog-th/Database/Groundhog/TH/Settings.hs

```diff
@@ -32,9 +32,10 @@ where
 import Control.Applicative
 import Control.Monad (forM, mzero, when)
 import Data.Aeson
+import Data.Aeson.Key (fromText)
+import qualified Data.Aeson.KeyMap as KeyMap
```

Comment:
In case it is helpful, here is how Amazonka writes compatible code between aeson 1 and aeson 2: https://github.com/brendanhay/amazonka/blob/3e66cc1b07f2818d0b4fc346310ecdb1c3c27599/lib/services/amazonka-dynamodb/src/Amazonka/DynamoDB/Types/AttributeValue.hs#L26-L30